### ISSUE TITLE
Clean Install of v9.10.00 RC2 has dupe BindingRedirect for System.Web.Http

### ISSUE THREAD

**Issue by jeremy-farrance** [dnnsoftware/Dnn.Platform#4748]
## Description of bug
After a clean install, checking web.config immediately after logging in as host for the first time, I noticed something unusual in web.config, search for, "System.Web.Http" and notice that there are 2 different binding redirects for the same name/publicKeyToken.

First one is at line 263:
```
<dependentAssembly>
  <assemblyIdentity name="System.Web.Http" publicKeyToken="31BF3856AD364E35" />
  <bindingRedirect oldVersion="0.0.0.0-5.2.3.0" newVersion="5.2.3.0" />
</dependentAssembly>
```
Second one is at line 295:
```
<dependentAssembly xmlns="urn:schemas-microsoft-com:asm.v1">
  <assemblyIdentity name="System.Web.Http" publicKeyToken="31bf3856ad364e35" />
  <bindingRedirect oldVersion="0.0.0.0-32767.32767.32767.32767" newVersion="5.2.7.0" />
</dependentAssembly>
```

Also noted that of the binding redirects updated (related to 5.2.7), only the System.Web.Http one was missing the usual comments about it having been upgraded. E.g.
```
<!--Upgraded by DotNetNuke.AspNetMvc version 5.2.61128.0 - Date: 7/15/2021 6:08:33 PM-->
<!--<dependentAssembly xmlns="urn:schemas-microsoft-com:asm.v1"><assemblyIdentity name="System.Web.Mvc" publicKeyToken="31bf3856ad364e35" /><bindingRedirect oldVersion="0.0.0.0-5.1.0.0" newVersion="5.1.0.0" /></dependentAssembly>-->
```

## Expected behavior
After the package upgrade runs (related to #4641), there should only be 1 binding redirect for the DLL.

## Affected version
* [x] 09.10.00 RC2 

### Various comments from the Slack discussion:
Mitchel Sellers: "That ain’t right"
Brian Dukes: "Not intentional," "We're probably doing a case sensitive search on public key to find existing binding redirects"
David Poindexter: "I smell an RC3"

**Comment by mitchelsellers** [comment]
Just confirming no third-party modules installed?

**Comment by jeremy-farrance** [comment]
Correct, as I said above, this is done immediately after the install and logging in for the first time. Not even a page refresh.

**Comment by david-poindexter** [comment]
Closing per #4749 

I also tested to ensure 2sxc 11.22.01 works on the build associated with that PR and it does.  There was a small glitch when placing a module on the page, but I think that is completely unrelated.  Ch0osing to "complete upgrade" within 2sxc (even though it was a fresh install of 11.22.01) resolved the issue.  Everything worked as expected at that point.

Thanks @jeremy-farrance for reporting this issue!
